plugins/manipulator.cpp

```diff
@@ -285,14 +297,14 @@ struct UnitInfo
     bool allowEdit;
     string name;
     string transname;
+    string lastname;
```

Comment:
Can't you just get this from unit->name?

Reply:
Maybe i looked and anorher issue, i mean - cant you get name from unit->name too? yet name is in uinfo here. I cached last name here because it is accessed by a sort comparator.